modules/android/partials/release-notes/couchbase-mobile-android-release-note.3.1.8.adoc

```diff
@@ -0,0 +1,21 @@
+[#maint-3-1-8]
+== 3.1.8 -- May 2024
+
+Version 3.1.8 for {param-title} delivers the following features and enhancements:
+
+NOTE: For an overview of the latest features offered in Couchbase Lite 3.1, see xref:ROOT:cbl-whatsnew.adoc[New in 3.1]
```

Comment:
Consider moving this note...it's a little jarring to see it right after your intro sentence.

Reply:
Moved it to the foot of release notes and applied this change to all of them across the platforms. Looks much better thank you! 👍🏼